spec/spec_helper.rb

```diff
@@ -38,6 +38,13 @@
   config.default_sleep_interval = 0.33
   config.clear_lets_on_failure = true
   config.default_retry_count = 3
+
+  unless ENV['FULL_BACKTRACE']
```

Comment:
That'd be nice - can you provide an example of how we can do that? I think you're better at rspec than I am :smile: